tests/functionTests.php

```diff
@@ -0,0 +1,77 @@
+<?php
+
+use PHPUnit\Framework\TestCase;
+
+require ('../functions.php');
+
+class StackTest extends TestCase
+{
+
+    /*
+     * This success test checks whether the function returns a string as expected
+     */
+    public function testArrayToTextSuccess()
+    {
+        $expected = "Some words in a string";
+        $example = [0 => ["key" => "Some words in a string"]];
+        $example2 = "key";
+        $case = arrayToText($example, $example2);
+        $this->assertEquals($case, $expected);
+    }
+
+    /*
+     * This failure test checks whether the function returns a string as expected
+     */
+    public function testArrayToTextFailure()
+    {
+        $expected = "this isn't an array";
+        $example = ["key" => "Some words in a string"];
+        $example2 = "key";
+        $case = arrayToText($example, $example2);
+        $this->assertEquals($case, $expected);
+    }
+
+    /*
+     * This success test checks whether the function returns a string of html
+     */
+    public function testProduceProjectSuccess()
+    {
+        $expected = '<div class="pilotShopBox"><h3>title</h3><p>fillText</p><a href=url>CLICK HERE to view</a></div>';
```

Comment:
this should tell you that the class `pilotShopBox` is not a good choice, as the function can produce any project